apps/webapp/src/state/send.ts

```diff
@@ -97,7 +97,7 @@ const assembleRequest = async ({ amount, recipient, selection, memo }: SendSlice
     ],
     source: new AddressIndex({ account: selection.accountIndex }),
     memo: new MemoPlaintext({
-      returnAddress: await getAddressByIndex(selection.accountIndex),
+      returnAddress: await getEphemeralAddress(selection.accountIndex),
```

Comment:
Wait, why this change? We always want sends to be from an ephemeral address?

Reply:
i'm guessing this was a rebase or something, but we never want to use ephemeral addresses here